modules/RTC/RTCUtils.js

```diff
@@ -1338,8 +794,11 @@ class RTCUtils extends Listenable {
             const audioTracks = avStream.getAudioTracks();
 
             if (audioTracks.length) {
-                const audioStream = new MediaStream(audioTracks);
+                const audioStream = new MediaStream();
 
+                for (let i = 0; i < audioTracks.length; i++) {
```

Comment:
why is this change needed? is that media stream constructor not supported on some browsers? if so, please leave a comment.

Reply:
The media stream constructor doesn't seem to work when we use MediaStreamTrackMock(unit tests) instead of MediaStreamTrack.

Reply:
Aha. Maybe leave a comment? I wouldn't have guessed :-)

Reply:
Sure, will do.